lib/sisimai/bite/email/x1.rb

```diff
@@ -0,0 +1,124 @@
+module Sisimai::Bite::Email
+  module X1
+    # Sisimai::Bite::Email::X1 parses a bounce email which created by Unknown
+    # MTA #1. Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/X1.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
+        :from    => %r/["]Mail Deliver System["] /,
+        :subject => %r/\AReturned Mail: /,
+      }
+      Re1 = {
+        :begin   => %r/\AThe original message was received at (.+)\z/,
+        :endof   => %r/\A__END_OF_EMAIL_MESSAGE__\z/,
+        :rfc822  => %r/\AReceived: from \d+[.]\d+[.]\d+[.]\d/,
+      }
+      Indicators = Sisimai::Bite::Email.INDICATORS
+
+      def description; return 'Unknown MTA #1'; end
```

Comment:
__[rubocop]__
* Style/SingleLineMethods: Avoid single-line method definitions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-single-line-methods)
* Style/Semicolon: Do not use semicolons to terminate expressions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-semicolon)
* Layout/TrailingWhitespace: Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

<!-- Comment by [SideCI](https://sideci.com) -->